parquet-column/src/main/java/org/apache/parquet/io/api/Binary.java

```diff
@@ -613,4 +681,60 @@ private static final boolean equals(byte[] array1, int offset1, int length1, byt
     }
     return true;
   }
+
+  // TODO java-doc
```

Comment:
Please, do not leave TODOs in the final code. (javadoc comments wouldn't be required for private methods anyway.)

Reply:
Sorry, didn't push the latest commit. This is fixed.